src/test/groovy/com/getkeepsafe/dexcount/DexFileTest.groovy

```diff
@@ -0,0 +1,22 @@
+package com.getkeepsafe.dexcount;
+
+import spock.lang.Specification
+
+class DexFileTest extends Specification {
+    def "test AAR dexcount"() {
+        setup:
+        def currentDir = new File(".").getAbsolutePath()
+
+        when:
+        def aarFile = new File(currentDir + File.separatorChar + "src"
+                + File.separatorChar + "test" + File.separatorChar + "resources"
+                + File.separatorChar + "android-beacon-library-2.7.aar")
```

Comment:
Where is this file?  It's not in the PR, did you forget to add it?

In any event, I'd prefer that we package this file as a test resource, ie. in `src/test/resources`.  That way, it gets included in the test .jar.  Self-contained tests FTW!